src/pfunit/core/MpiTestCase.F90

```diff
@@ -24,7 +25,7 @@
 !-------------------------------------------------------------------------------
 
 module PF_MpiTestCase
-   use mpi
+   use PF_MPI_MODULE
```

Comment:
I am a little bit concerned that this approach causes problems with CMake's dependency generator.   I know in the past I've had issues with projects where the module name is defined by the preprocessor.  Maybe this case is safe though?  Or CMake has improved to use the defines that are known?

Reply:
I'm reasonably confident that is safe -- CMake is now pretty good with preprocessing before doing the dependency tree.

If not, the below might be better:
```fortran
#ifdef PF_USE_MPI_F08
    use mpi_f08
#else
    use mpi
#endif
```

Reply:
ok - I'll keep that in mind.  For now your changes pass the regression tests, so probably good as you said.